backend/core/src/listings/types/listing-status-enum.ts

```diff
@@ -1,5 +1,6 @@
 export enum ListingStatus {
   active = "active",
-  pending = "pending",
   closed = "closed",
+  new = "new",
```

Comment:
What does `new` represent as a status?

Reply:
It's an initial state for the state machine. Pending state requires already some validations to pass, so we need an initial point.